Tools/Scripts/configure-xcode-for-embedded-development

```diff
@@ -227,7 +227,11 @@ def do_copy(source_path, dest_path):
         if dest_path.exists():
             shutil.rmtree(dest_path)
         ensure_parent_exists(dest_path)
-        shutil.copytree(source_path, dest_path)
+        try:
+            shutil.copytree(source_path, dest_path)
+        except:
```

Comment:
Not a problem if we only do this for one line, but if we start making this try-except bigger, it's worth being more specific about the exception, this kind of try/except will suppress many errors which would be considered compiler errors in other languages.